content/docs/how-to-guides/connectivity-issues.md

```diff
@@ -7,20 +7,20 @@ title: Connecting with old clients
 In most cases, copy-pasting `Connection string` from the project's dashboard and using it in your project should work as is. However, with older clients and some native Postgres clients, you may receive the following error:
 
 ```txt
-ERROR: Project name is not specified. Either please upgrade the postgres client library (libpq) for SNI support or pass the project name as a parameter: '&options=project%3D<project-name>'. See more at https://neon.tech/sni
+ERROR: Project ID is not specified. Either please upgrade the postgres client library (libpq) for SNI support or pass the project ID (first part of the domain name) as a parameter: '&options=project%3D<project-id>'. See more at https://neon.tech/sni
```

Comment:
ERROR: The project ID is not specified. Either upgrade the PostgreSQL client library (libpq) for SNI support or pass the project ID (the first part of the domain name) as a parameter: '&options=project%3D<project-id>'. See https://neon.tech/sni for more information.